src/Select.js

```diff
@@ -487,7 +487,7 @@ export default class Select extends Component<Props, State> {
 
   openMenu(focusOption: 'first' | 'last') {
     const { menuOptions, selectValue } = this.state;
-    const { isMulti } = this.props;
+    const { isMulti, disableAutoFocus } = this.props;
```

Comment:
missed flow type

Reply:
Even the documentation is missing but I needed a confirmation if they're willing to merge this new feature.

Reply:
Merge please.

Reply:
It seems there is now a conflict in this merge request